public/angular/js/controllers/media/settings/settings_modal.js

```diff
@@ -0,0 +1,49 @@
+angular.module('Aggie')
+
+.controller('MediaSettingsModalInstanceController', [
+  '$scope',
+  '$modalInstance',
+  'Settings',
+  'FlashService',
+  'settingsValues',
+  'media',
+  '$filter',
+  function($scope, $modalInstance, Settings, flash, settings, media, $filter) {
+
+    $scope.settings = settings;
+    $scope.media = media;
+
+    $scope._showErrors = false;
```

Comment:
Why is there a blank line above here? Seems the blank should instead be below. Please try to group code into logical 'paragraphs'.